srv/salt/ceph/stage/prep/minion/default.sls

```diff
@@ -6,6 +6,12 @@ crc_method minion:
     - tgt_type: compound
     - sls: ceph.salt.crc.minion
 
+set apparmor:
```

Comment:
That also needs to be added in all the non-default .sls files

Reply:
sigh...yes. Reorganized the common parts into a common.sls, that should help in the future.